plugins/JsTrackerInstallCheck/lang/en.json

```diff
@@ -0,0 +1,8 @@
+{
+    "JsTrackerInstallCheck": {
+        "TestInstallationDescription": "Click the button below to test the tracking code installation. A new tab will open, and should close within 10 seconds.",
+        "TestInstallationBtnText": "Test installation",
+        "JsTrackingCodeInstallCheckSuccessMessage": "The tracking code is installed successfully! This screen will disappear as soon as some data is tracked for your website.",
+        "JsTrackingCodeInstallCheckFailureMessage": "Tracking code verification failed. Please verify that you have installed the code correctly; if you need help or have any questions please contact our support team."
```

Comment:
This message might need to be reworked. On premise we never suggest to contact support. If we have some useful guide around this topic we can link that instead maybe.

Reply:
This is the text that Product came up with while I was working through the design with them. What do you think @Stan-vw ? Should we rework the part that mentions contacting support?

Reply:
I chatted with Stan and we're going to remove the portion about contacting support. Thank you for raising that @sgiehl